app/models/__init__.py

```diff
@@ -6,3 +6,4 @@
 from agency import *  # noqa
 from user import *  # noqa
 from incident_report import *  # noqa
+from miscellaneous import *  # noqa
```

Comment:
why `miscellaneous` and not `editableHTML`? what other misc classes would potentially be here?